src/frontend/app.ts

```diff
@@ -97,6 +97,17 @@ export default (octokitApp: App): Express => {
 
 	app.use(logMiddleware);
 
+	// Catch non successful responses
+	app.use((req, res, next) => {
+		res.once("finish", () => {
+			// if status is under 200 or higher/equal to 400, but not 503 in maintenance mode, log it to figure out issues
+			if ((res.statusCode < 200 || res.statusCode >= 400) && !(res.statusCode === 503 && booleanFlag(BooleanFlags.MAINTENANCE_MODE, false))) {
```

Comment:
Do we still need maintenance mode?

Reply:
It's to ignore maintenance mode 503s

Reply:
Yes, but would we need it at all? We've already performed the migration, so eventually we'll cleanup the flag.